run/test/specs/input_validations/onboarding_wrong_seed.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { androidIt, iosIt } from '../../../types/sessionIt';
+import * as onboarding from '../locators/onboarding';
+import { openAppOnPlatformSingleDevice, SupportedPlatformsType } from '../utils/open_app';
+import { localize } from '../../../localizer/i18n/localizedString';
+
+iosIt('Onboarding wrong seed', onboardingIncorrectSeed);
+androidIt('Onboarding wrong seed', onboardingIncorrectSeed);
+
+// the seed phrase is too long and we expect to get the generic error 
+const wrongSeed = 'ruby bakery illness push rift reef nabbing bawled hope ruby silk lobster hope ruby ruby ruby'
+// the expected error is 'Please check your recovery password' which is represented by the following localized string
+const expectedError = localize('recoveryPasswordErrorMessageGeneric').strip().toString();
+
+
+async function onboardingIncorrectSeed(platform: SupportedPlatformsType) {
+    const { device } = await openAppOnPlatformSingleDevice(platform);
+    await device.clickOnElementAll(new onboarding.AccountRestoreButton(device));
```

Comment:
do you have a suggestion? I quite liked the simplicity of `import * as onboarding` but I guess I can just specify the imports with `import { X, Y, Z } from '../locators/onboarding';` ?

Reply:
I agree that the import * is nice and clean but its preferable to make sure we're not locking locators into specific pages, just in case we start making the components more generic, and therefore reusable.